build/common.project.props

```diff
@@ -31,6 +31,7 @@
     <XunitConsoleExePath>$(SolutionPackagesFolder)xunit.runner.console.2.3.1\tools\net452\xunit.console.x86.exe</XunitConsoleExePath>
     <ILMergeExePath>$(SolutionPackagesFolder)ILMerge.2.14.1208\tools\ILMerge.exe</ILMergeExePath>
     <XunitXmlLoggerDirectory>$(SolutionPackagesFolder)XunitXml.TestLogger.2.0.0\build\_common</XunitXmlLoggerDirectory>
+    <NuGetBuildTasksPackTargets>$(SolutionPackagesFolder)NuGet.Build.Tasks.Pack.4.9.2\build\NuGet.Build.Tasks.Pack.targets</NuGetBuildTasksPackTargets>
```

Comment:
why can't we just use the one from the current build?

Reply:
Kind of breaks the whole repeatability thing. And it only works with build.ps1, not necessarily if you want to pack individual projects.
It also will have file access issues when you are trying to pack the pack tasks package itself.

Reply:
I'm not able to follow the repeatability concerns here. What exactly it breaks? why shouldn't this always pack our projects with in-built pack target? It will also add an additional check if our pack target breaks something for normal cases...

Also, not able to see through other concerns, may be it'll be better to talk offline and get clarity.

Reply:
Discussed offline, we're good here.